src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers, createStore } from 'redux';
+import books from './books/books';
+import categories from './categories/categories';
+
+const root = combineReducers({ books, categories });
+
+const store = createStore(
+  root,
+  {},
+);
+
+export default store;
```

Comment:
* You used `combineReducers` to create a reducer 🙌
* You used `createStore` to create a store through the reducer created 🙌

Reply:
Hello, @mateo951 Thanks for the code review. 
I'm not sure I understand.
 The prerequisite I require a detailed description of what has to be changed in my project.

Reply:
@mateo951  I now get what you require me to change, and now I have made all changes possible you can now check.

Reply:
Hey @Alfredbis29, sorry for the late response! Happy to know everything went well 🔥